### PR TITLE
yggdrasil: init at 0.3.5

### PR DESCRIPTION
**Pull request by Lassulus**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/networking/yggdrasil/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "ygdrassil";
```
Comment (gazally) [inline pkgs/tools/networking/yggdrasil/default.nix]:
```suggestion
  pname = "yggdrasil";
```

**Review by gazally — COMMENTED** [review]
Hi! I've been trying out `yggdrasil` lately too.  The derivation I wrote uses `buildGoModule` and I also have a NixOS service module and I will put both into a pull request shortly so you can see them. 
 
I just built your derivation on my system and it works for me.

**Comment by Lassulus** [comment]
@gazally cool, do you want to rebase your service on this package or should we wait?
I'm also not sure what buildGoModule does in this context.

**Comment by Lassulus** [comment]
I will close this in favor of #63863

**Pull request closed without merging**